app/src/main/java/fi/aalto/legroup/achso/entities/Annotation.java

```diff
@@ -16,8 +16,7 @@
     protected String text;
     protected User author;
 
-    @SuppressWarnings("UnusedDeclaration")
-    private Annotation() {
+    public Annotation() {
```

Comment:
Looks like the code that uses empty constructors (`Annotation()`, `Video()`, `OptimizedVideo()`...) is inside the `entities` package.

Could it make sense to mark these empty constructors package-private so that it’s clear that these should only be used in that way?